packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -86,19 +95,48 @@ function FlowsheetDataGrid(props) {
     rowHeight,
     onCellSelect,
     onClearSelectedCells,
+    intl,
   } = props;
 
+  const flowsheetColumns = useMemo(() => columns.map(column => ({ ...column, isResizable: false })), [columns]);
+  const pinnedColumns = flowsheetColumns.length ? [flowsheetColumns[0]] : [];
+  const overflowColumns = flowsheetColumns.length > 1 ? flowsheetColumns.slice(1) : [];
+
+  const flowsheetRows = useMemo(() => {
+    const noResultCellContent = (
+      <>
+        <span aria-hidden>{intl.formatMessage({ id: 'Terra.flowsheetDataGrid.no-result-display' })}</span>
+        <VisuallyHiddenText text={intl.formatMessage({ id: 'Terra.flowsheetDataGrid.no-result' })} />
+      </>
+    );
+
+    const newRows = rows;
```

Comment:
You still need to clone on this line and the new cells line to not impact the source prop.
const newRows = [...rows]
const newCells = [...cells]

Reply:
Good catch, fixed